src/wiring/fetchConnectPage.ts

```diff
@@ -0,0 +1,31 @@
+import { Page } from "playwright";
+import { stringify } from "qs";
+
+export interface FetchConnectorParams {
+  book: string;
+  vehicleId: string;
+  // cell: string;
+  country: string;
+  bookType: string;
+  language: string;
+  item: string;
+  // title: string;
+}
+
+export default async function fetchConnectorPage(
+  params: FetchConnectorParams,
+  browserPage: Page,
+  saveImagePath: string
+): Promise<void> {
+  const url = `https://www.fordtechservice.dealerconnection.com/Wiring/face/?${stringify(
+    params
+  )}`;
+  console.log(url);
```

Comment:
No worries in leaving a log throughout the code-- I have a couple-- but would you mind formatting the URL, something like 
```suggestion
  console.log(`Downloading wiring connector page ${url}`);
```